CMakeLists.txt

```diff
@@ -91,29 +91,36 @@ elseif(KernelArchRiscV)
         # => can't change that easily without messing up the build system
         execute_process(COMMAND ${CROSS_COMPILER_PREFIX}as --version OUTPUT_VARIABLE _gas_version)
         # Extract version number
-        string(REGEX
-            MATCH
```

Comment:
nitpicking: can you make this syle change a separate commit?

Reply:
As the binutils version check will be removed in favor of the GCC version check alone, this style change is also gonna disappear in the process.